defaults/main.yml

```diff
@@ -0,0 +1,31 @@
+harbor_data_dir: "/data"
+harbor_cert_dir: "{{ harbor_data_dir }}/cert"
+domain_name: "{{ ansible_host }}"
+
+harbor_admin_password: "{{ vault_harbor_admin_password }}"
+harbor_db_password: "{{ vault_harbor_db_password }}"
+harbor_core_secret: "{{ vault_harbor_core_secret }}"
+
+vault_harbor_admin_password: !vault |
```

Comment:
not requires to add here
instead create vault specific file and include it when run